cmd/certs/main.go

```diff
@@ -46,10 +47,11 @@ var (
 )
 
 type config struct {
-	LogLevel  string `env:"MF_CERTS_LOG_LEVEL"        envDefault:"info"`
-	CertsURL  string `env:"MF_SDK_CERTS_URL"          envDefault:"http://localhost"`
-	ThingsURL string `env:"MF_THINGS_URL"             envDefault:"http://things:9000"`
-	JaegerURL string `env:"MF_JAEGER_URL"             envDefault:"localhost:6831"`
+	LogLevel   string `env:"MF_CERTS_LOG_LEVEL"        envDefault:"info"`
+	CertsURL   string `env:"MF_SDK_CERTS_URL"          envDefault:"http://localhost"`
+	JaegerURL  string `env:"MF_JAEGER_URL"             envDefault:"localhost:6831"`
+	ThingsURL  string `env:"MF_THINGS_URL"             envDefault:"http://things:9000"`
+	InstanceID string `env:"MF_CERTS_INSTANCE_ID"   envDefault:""`
```

Comment:
envDefault of InstanceID is not algined with others


```suggestion
	JaegerURL  string `env:"MF_JAEGER_URL"             envDefault:"localhost:6831"`
	ThingsURL  string `env:"MF_THINGS_URL"             envDefault:"http://things:9000"`
	InstanceID string `env:"MF_CERTS_INSTANCE_ID"      envDefault:""`
```